manager/orchestrator/replicated/services.go

```diff
@@ -198,10 +198,23 @@ func (r *Orchestrator) addTasks(ctx context.Context, batch *store.Batch, service
 	}
 }
 
-func (r *Orchestrator) deleteTasks(ctx context.Context, batch *store.Batch, slots []orchestrator.Slot) {
+// setTasksDesiredState sets the desired state of all tasks to what
+// is requested.
```

Comment:
sets the desired state of all tasks to what is requested => sets the desired state for all tasks *for the given slots* to the given state.